en/django_installation/instructions.md

```diff
@@ -165,6 +165,8 @@ In the console, run `pip install --upgrade pip`.
 
 Then run `pip install django~=1.9.0` (note that we use a tilde followed by an equal sign: `~=`) to install Django.
 
+**NOTE:** When you see command-line examples like the following you only need to type what is written after the `$` character (or `>` on Windows). The other lines represents the output of the command you entered.
```

Comment:
> [...] you should probably put this the first time they encounter the command line. It's in python instruction file.

Actually, the first time is even before that: in the "Introduction to the command-line interface" chapter ([book](https://tutorial.djangogirls.org/en/intro_to_command_line/#your-first-command-yay) / [source](https://github.com/DjangoGirls/tutorial/tree/a26bcdaf536f144ec6d17d0808c3d2bf64137db9/en/intro_to_command_line#your-first-command-yay))